File: scratch/replace_text_center.py
import os

def fix_file(filepath):
    with open(filepath, 'r', encoding='utf-8') as f:
        lines = f.readlines()
    
    changed = False
    for i, line in enumerate(lines):
        if '<th className="py-2' in line or '<td className="py-2' in line:
            if 'text-left' in line or 'text-right' in line:
                new_line = line.replace('text-left', 'text-center').replace('text-right', 'text-center')
                lines[i] = new_line
                changed = True
                
    if changed:
        with open(filepath, 'w', encoding='utf-8') as f:
            f.writelines(lines)
        print(f"Fixed {filepath}")
    else:
        print(f"No changes for {filepath}")

fix_file(r"c:\Users\ferna\Downloads\salon-suite-pro\AppSalaodeBeleza\screens\SalonComissoesDashboard.tsx")
fix_file(r"c:\Users\ferna\Downloads\salon-suite-pro\AppSalaodeBeleza\collaborator-app\src\screens\SalonComissoesDashboard.tsx")

File: scratch/replace_text_center.js
const fs = require('fs');

function fixFile(filepath) {
    let content = fs.readFileSync(filepath, 'utf-8');
    let lines = content.split('\n');
    let changed = false;

    for (let i = 0; i < lines.length; i++) {
        let line = lines[i];
        if (line.includes('<th className="py-2') || line.includes('<td className="py-2')) {
            if (line.includes('text-left') || line.includes('text-right')) {
                lines[i] = line.replace(/text-left/g, 'text-center').replace(/text-right/g, 'text-center');
                changed = true;
            }
        }
    }

    if (changed) {
        fs.writeFileSync(filepath, lines.join('\n'), 'utf-8');
        console.log('Fixed ' + filepath);
    } else {
        console.log('No changes for ' + filepath);
    }
}

fixFile('c:\\\\Users\\\\ferna\\\\Downloads\\\\salon-suite-pro\\\\AppSalaodeBeleza\\\\screens\\\\SalonComissoesDashboard.tsx');
fixFile('c:\\\\Users\\\\ferna\\\\Downloads\\\\salon-suite-pro\\\\AppSalaodeBeleza\\\\collaborator-app\\\\src\\\\screens\\\\SalonComissoesDashboard.tsx');
